apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -34,7 +34,7 @@ export function ExecutionDetailsModal({
       styles={{
         modal: {
           backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
-          width: '60%',
+          width: '90%',
```

Comment:
In the designs the width of the modal was set to a 60% of the screen size. I guess it is ok if we increase it.

Reply:
Yeah this was needed to make the payload look better in the screenshots...